bytescount_client/stats_sender.go

```diff
@@ -9,11 +9,11 @@ import (
 
 type SessionStatsSender func(bytesSent, bytesReceived int) error
 
-func NewSessionStatsSender(mysteriumClient server.Client, sessionId session.SessionId, signer identity.Signer) SessionStatsSender {
-	sessionIdString := string(sessionId)
+func NewSessionStatsSender(mysteriumClient server.Client, sessionID session.SessionID, signer identity.Signer) SessionStatsSender {
```

Comment:
Are You using high linter requirements? What command are You using?

Reply:
`bin/lint` - which hides less important errors.